pages/domains/domains_choosing.md

```diff
@@ -104,42 +103,41 @@ Examples:
 ### Counties
 This organization type includes counties, parishes, or boroughs.
 
-Most county .gov domains must include the two-letter state abbreviation or the full state name. County names that aren’t shared by any other city, county, parish, town, borough, village or equivalent in the U.S., at the time a domain is granted, can be requested without referring to the state. Counties can include “county” in their domain to distinguish it from other places with similar names. We use the [Census Bureau’s National Places Gazetteer Files](https://www.census.gov/geographies/reference-files/time-series/geo/gazetteer-files.html){.usa-link--external} to determine if county names are unique.
+Most county .gov domains must include the two-letter state abbreviation or the full state name. County names that aren’t shared by any other city, county, parish, town, borough, village or equivalent in the U.S., (at the time a domain is granted) don’t have to refer to their state in their domain name. Counties can include “county” in their domain to distinguish it from other places with similar names. 
```

Comment:
> equivalent in the U.S., (at the time a domain is granted)

Can we move the comma to the end of the parentheses?

Reply:
Moved 🚗

Reply:
@michelle-rago @h-m-f-t I'm actually realizing that the comma should have never been there to begin with. It's not in the google doc version of this page, or in the request form, where this same content appears.